docs/easybuild-v5/enhancements.md

```diff
@@ -57,3 +58,16 @@ To disable trace output, either:
     [override]
     trace=0
     ```
+
+---
+
+## Require `download_instructions` to be specified {: require_download_instructions }
+
+[`download_instructions`][download_instructions] is used in easyconfigs to specify instructions, or information, on
+how to obtain sources that are not directly downloadable. In EasyBuild 5, we require that easyconfigs specify these
```

Comment:
That's not actually entirely true, we only require this in the easyconfigs test suite, *not* in EasyBuild itself...

That's big difference, so we should reword this imho...